### PR TITLE
add a new web scrapper

### PR DESCRIPTION
**Pull request by Jaideep07**
# Description

This pull request contains a new web scrapper

Fixes #(145)

Replace `145` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [X ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X ] I have named my files and folder, according to this project's guidelines.
- [X ] My code follows the style guidelines of this project.
- [X ] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X ] I have commented on my code, particularly in hard-to-understand areas.
- [X ] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [ X] I have included a requirements.txt file (if external libraries are required.)
- [X ] My changes do not produce any warnings.
- [X ] I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Scripts/Web_Scrappers/Rotten_Tomatoes_Scrapper/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bs4
+requests
```
Comment (AdityaJ7) [inline Scripts/Web_Scrappers/Rotten_Tomatoes_Scrapper/requirements.txt:2]: replace it with  - requests==2.24.0

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]

**Comment by Jaideep07** [comment]
Done 👍

**Comment by AdityaJ7** [comment]
The dash '-' was just for indication purpose

**Comment by AdityaJ7** [comment]
@all-contributors add @Jaideep07 for code

**Pull request merged**